README.rst

```diff
@@ -50,10 +50,10 @@ environment variables ::
 
 Then run the following commands to bootstrap your environment ::
 
-    git clone https://github.com/hndrewaall/league
+    git clone https://github.com/massgo/league
```

Comment:
You should actually be cloning your personal fork, so maybe `<your github username>` would be better?